File: app/src/main/java/com/cangwang/modulebus/ExModule/BaseController.java
package com.cangwang.modulebus.ExModule;

import android.app.Activity;
import android.content.Context;
import android.os.Bundle;
import android.view.View;
import android.view.ViewGroup;

/**
 * Created by cangwang on 2017/6/15.
 */

public class BaseController implements IView{
    private Context context;

    public void onAttach(Activity activity){
        context= activity;
    }

    public void create(Bundle saveInstanceState, Activity activity,ViewGroup container){

    }

    @Override
    public View getView() {
        return null;
    }

    @Override
    public void hide() {

    }

    @Override
    public void show() {

    }
}

File: app/src/main/java/com/cangwang/modulebus/ExModule/BaseController.kt
package com.cangwang.modulebus.ExModule

import android.app.Activity
import android.content.Context
import android.os.Bundle
import android.view.View
import android.view.ViewGroup

/**
 * Created by cangwang on 2017/6/15.
 */
class BaseController : IView {
    private var context: Context? = null
    fun onAttach(activity: Activity?) {
        context = activity
    }

    fun create(saveInstanceState: Bundle?, activity: Activity?, container: ViewGroup?) {}
    override val view: View?
        get() = null

    override fun hide() {}
    override fun show() {}
}